.gitignore

```diff
@@ -16,8 +16,10 @@
 npm-debug.log*
 yarn-error.log
 testem.log
+.DS_Store
```

Comment:
I literally just mentioned this in another PR earlier today (😆), but I'm personally of the opinion that since `.DS_Store` is a system-specific file and not a project-specific file, it belongs in a user's `~/.gitignore` and not in the project `.gitignore`

Reply:
I can be sold on this. @twokul @mixonic any strong feelings one way or the other?